### PR TITLE
fix comment error in mnist_softmax.py

### PR DESCRIPTION
**Pull request by MrQianjinsi**
should be tf.nn.softmax

### CODE REVIEW

**Comment by MrQianJinSi** [comment]
I signed it! please check.

**Review by rohan100jain — APPROVED** [review]

**Comment by rohan100jain** [comment]
@tensorflow-jenkins test this please
